src/main/java/frc/robot/RobotContainer.java

```diff
@@ -76,7 +79,11 @@ public RobotContainer() {
     simulationSubsystem = Robot.isSimulation() && swerveSubsystem != null ? new SimulationSubsystem(swerveSubsystem) : null;
     subsystemEnabledLog.append(simulationSubsystem == null ? "Simulation: Disabled" : "Simulation: Enabled");
 
-    intakeSubsystem = Robot.intakeEnabled ? new IntakeSubsystem() : null;
+    if (Robot.intakeEnabled && Robot.tofEnabled) {
+      intakeSubsystem = new IntakeSubsystem(timeOfFlightSensor);
+    } else {
+      intakeSubsystem = Robot.intakeEnabled ? new IntakeSubsystem() : null;
```

Comment:
The ternary was clearer when it was it's own line.  

However, I think it would be clearer to have else if Robot.intakeEnabled  

Or change to be 
if Robot.intakeEnabled  for the first if
  if Robot.tofEnabled
    intakesubsystem = new intake(tof)
  else
    intakesubsystem = new intake()
else
  intakesubsysem = nuill